app/src/main/java/org/kiwix/kiwixmobile/downloader/DownloadService.java

```diff
@@ -176,6 +180,7 @@ public int onStartCommand(Intent intent, int flags, int startId) {
   }
 
   public void stopDownload(int notificationID) {
+    Log.i("kiwixdownloadservice", "Stopping ZIM Download");
```

Comment:
I'm in two minds whether it'd be useful to log the notificationID. Sometimes the service will be downloading several files in parallel. It might be useful to identify which download is being stopped. I don't yet know whether we have enough context to (a) know how many downloads are active currently? (b) know which download is being stopped (i.e. which ZIM file) and what percentage has been downloaded when the stop was requested...